scripts/code_size_compare.py

```diff
@@ -33,7 +33,8 @@
 class CodeSizeComparison:
     """Compare code size between two Git revisions."""
 
-    def __init__(self, old_revision, new_revision, result_dir):
+    def __init__(self, old_revision, new_revision, result_dir,
+                 config=None, symbols=False, buildtag=None):
```

Comment:
To satisfy pylint:
```suggestion
                 #pylint: disable=too-many-arguments
                 config=None, symbols=False, buildtag=None):
```
(or any other way to typeset the pylint directive)

> I don't think this caused an error, or did it? The overall rating is still 9.9/10.

We don't distinguish between the severities of pylint diagnostics. It's pass or fail.

Reply:
Got it. I followed your suggestion to silence pylint.